README.md

```diff
@@ -31,5 +31,5 @@ After installing the software, you can run the program on mat files generated by
     afqbrowser-assemble afq.mat
     afqbrowser-run
 
-Open a browser pointing to http://localhost:8080, to view the visualization of
```

Comment:
Maybe we should set the default to 8080? 8888 is usually what Jupyer chooses per default, and we might want to set up machines that can do both (e.g., on AWS). 

We'd probably also need to change here: https://github.com/yeatmanlab/AFQ-Browser/blob/master/afqbrowser/browser.py#L147